packages/app-mobile/components/FolderPicker.tsx

```diff
@@ -0,0 +1,90 @@
+const React = require('react');
+
+import { FunctionComponent } from 'react';
+import { _ } from '@joplin/lib/locale';
+import Setting from '@joplin/lib/models/Setting';
+import Folder, { FolderEntityWithChildren } from '@joplin/lib/models/Folder';
+const { themeStyle } = require('./global-style.js');
+import Dropdown, { DropdownListItem, OnValueChangedListener } from './Dropdown';
+import { FolderEntity } from '@joplin/lib/services/database/types';
+
+interface FolderPickerProps {
+  disabled?: boolean;
+  selectedFolderId?: string;
+  onValueChange?: OnValueChangedListener;
+  mustSelect?: boolean;
+	folders: FolderEntity[];
+  placeholder?: string;
+  dark?: boolean;
+}
+
+
+const FolderPicker: FunctionComponent<FolderPickerProps> = ({
+	disabled,
+	selectedFolderId,
+	onValueChange,
+	mustSelect,
+	folders,
+	placeholder,
+	dark,
+}) => {
+	const themeId = Setting.value('theme');
```

Comment:
It might make sense to make `themeId` a prop — if we ever implement automatically changing the app theme to match the system theme, we'll want `themeId` to update automatically.

Note that making `themeId` a prop can help with auto-updating the theme in the FolderPicker because `ScreenHeader.tsx`'s `themeId` prop is managed by React Redux, and so should update with the app's theme: 
https://github.com/laurent22/joplin/blob/6aaea8ad4f13321f851c4d9f70ed462d7916458d/packages/app-mobile/components/ScreenHeader.tsx#L672

Reply:
Fixed in c156b9c